server/client/ruby/candlepin_api.rb

```diff
@@ -1227,6 +1236,11 @@ def post_text(uri, data=nil)
     return response.body
   end
 
+  def post_text_return_json(uri, data=nil)
```

Comment:
We should probably just pass in the response type as a method param and have it default to the accept/text. Another optional arg similar to data=nil.